json/src/lib.rs

```diff
@@ -2102,6 +2103,271 @@ pub struct MasternodeStatus {
+    #[serde(with = "::serde_hex")]
+    pub id: Vec<u8>,
+    #[serde(with = "::serde_hex")]
+    pub msg_hash: Vec<u8>,
+    #[serde(with = "::serde_hex")]
+    pub sign_hash: Vec<u8>,
+    #[serde(with = "::serde_hex")]
+    pub signature: Vec<u8>,
+}
+
+#[serde(untagged)]
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub enum QuorumSignResult{
+    QuorumSignStatus(bool),
+    QuorumSignSignatureShare(QuorumSignature),
+}
+
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct QuorumMemberOf {
+    pub height: u32,
+    #[serde(rename = "type")]
+    pub quorum_type: String,
+    pub quorum_hash: QuorumHash,
+    #[serde(with = "::serde_hex")]
+    pub mined_block: Vec<u8>,
+    #[serde(with = "::serde_hex")]
+    pub quorum_public_key: Vec<u8>,
+    pub is_valid_member: bool,
+    pub member_index: u32,
+}
+
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct QuorumMemberOfResult(
+    pub Vec<QuorumMemberOf>
+);
+
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct QuorumSnapshot {
+    pub active_quorum_members: Vec<bool>,
+    pub mn_skip_list_mode: u8,
+    pub mn_skip_list: Vec<u8>,
+}
+
+#[serde_as]
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct QuorumMasternodeListItem {
+    #[serde(with = "::serde_hex")]
+    pub pro_reg_tx_hash: Vec<u8>,
+    #[serde(with = "::serde_hex")]
+    pub confirmed_hash: Vec<u8>,
+    #[serde_as(as = "DisplayFromStr")]
+    pub service: SocketAddr,
+    #[serde_as(as = "Bytes")]
+    pub pub_key_operator: Vec<u8>,
+    #[serde_as(as = "Bytes")]
+    pub voting_address: Vec<u8>,
+    pub is_valid: bool,
+}
+
+#[serde_as]
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MasternodeListDiff {
+    pub base_block_hash: dashcore::BlockHash,
+    pub block_hash: dashcore::BlockHash,
+    #[serde_as(as = "Bytes")]
+    pub cb_tx_merkle_tree: Vec<u8>,
+    #[serde_as(as = "Bytes")]
+    pub cb_tx: Vec<u8>,
+    #[serde(rename = "deletedMNs")]
+    pub deleted_mns: Vec<QuorumMasternodeListItem>,
+    pub mn_list: Vec<QuorumMasternodeListItem>,
+    pub deleted_quorums: Vec<QuorumMinableCommitments>,
+    pub new_quorums: Vec<QuorumMinableCommitments>,
+    #[serde(rename = "merkleRootMNList", with = "::serde_hex")]
+    pub merkle_root_mn_list: Vec<u8>,
+    #[serde(rename = "merkleRootQuorums", with = "::serde_hex")]
+    pub merkle_root_quorums: Vec<u8>,
+}
+
+
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct QuorumRotationInfo {
+    pub extra_share: bool,
+    pub quorum_snapshot_at_h_minus_c: QuorumSnapshot,
+    pub quorum_snapshot_at_h_minus_2c: QuorumSnapshot,
+    pub quorum_snapshot_at_h_minus_3c: QuorumSnapshot,
+    pub mn_list_diff_tip: MasternodeListDiff,
```

Comment:
I think camel case will give something like MnListDiffTip when we need MNListDiffTip

Reply:
@QuantumExplorer camel case here gives `mnListDiffTip`, which is what's in the returned data as seen in the docs here:

<img width="377" alt="Screenshot 2022-09-08 at 17 51 41" src="https://user-images.githubusercontent.com/15247893/189180672-536f1e32-da47-4ca5-80ed-308a1d25db5b.png">

Reply:
Ah okay